index.js

```diff
@@ -79,11 +80,20 @@ module.exports = {
     }
   },
 
-  treeForAddon() {
-    // remove our "addon" folder from the build if we're stripping test selectors
-    if (!this._stripTestSelectors) {
-      return this._super.treeForAddon.apply(this, arguments);
-    }
+  treeForAddonTestSupport(tree) {
+    // intentionally not calling _super here
+    // so that can have our `import`'s be
+    // import testSelector from 'ember-test-selectors';
```

Comment:
_super would return same tree but with namespace including the sub folder -

`destDir: /${this.moduleName()}/test-support`

_super also returns already preprocessed js files.